docker-images/sw360empty/quickdeploy.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Copyright Siemens AG, 2019.
+# Part of the SW360 Portal Project.
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+
+set -e
+
+WATCH_DIR="$1"
+DEST_DIR="$2"
+LOGFILE="$3"
```

Comment:
Instead of writing `| tee -a "$LOGFILE"` everywhere you could add the line 
```
exec &> >(tee -a $LOGFILE)                                                                                                                                                                                        
```
once